docs/api-v1.json

```diff
@@ -0,0 +1,1136 @@
+          }
+        }
+      },
+      "ListingPostPut": {
+        "description": "Listing object used to create or update a listing (if published is set, following fields must be set on the listing: title, description, address, listing_type, available_to, available(auto set), publisher(auto set))",
+        "type": "object",
+        "properties": {
+          "id": {
+            "$ref": "#/components/schemas/UUID"
+          },
+          "creation_date": {
+            "$ref": "#/components/schemas/Date"
+          },
+          "title": {
+            "type": "string",
+            "description": "title of the listing",
+            "example": "beautiful apartment in Zurich",
+            "required": true
+          },
+          "description": {
+            "type": "string",
+            "description": "description of the listing",
+            "example": "beautiful apartment in Zurich with one bedroom and a balcony"
+          },
+          "address": {
+            "$ref": "#/components/schemas/Location"
+          },
+          "published": {
+            "type": "boolean",
+            "description": "if the listing is published",
+            "example": true,
+            "default": false
+          },
+          "sqm": {
+            "type": "number",
+            "description": "square meters of the listing",
+            "example": 45.5,
+            "required": false
+          },
+          "listing_type": {
+            "$ref": "#/components/schemas/ListingType"
+          },
+          "furnished": {
+            "type": "boolean",
+            "description": "if the listing is furnished",
+            "example": false,
+            "required": false
+          },
+          "available_to": {
+            "type": "array",
+            "description": "available to whom gender",
+            "items": {
+              "$ref": "#/components/schemas/Gender"
+            },
+            "example": [
+              "MALE",
+              "FEMALE",
+              "PREFER_NOT_TO_SAY",
+              "OTHER"
+            ]
+          },
+          "available": {
+            "type": "boolean",
+            "description": "if the listing is still available or not",
+            "example": true,
+            "required": false,
+            "default": true
+          },
+          "rent": {
+            "type": "number",
+            "description": "rent of the listing",
+            "required": false,
+            "example": 750
+          },
+          "deposit": {
+            "type": "number",
+            "description": "deposit of the listing",
+            "required": false,
+            "example": 2500
+          },
+          "rooms": {
+            "type": "number",
+            "description": "number of rooms of the listing",
+            "required": false,
+            "example": 2.5
+          },
+          "publisher": {
+            "$ref": "#/components/schemas/UserGetPublic"
```

Comment:
Isn't it only with the UUID of the user

Reply:
the get sends a public user version `private UserGetPublicDto publisher;`